neuralop/layers/neighbor_search.py

```diff
@@ -1,6 +1,14 @@
 import torch
 from torch import nn
 
+# only install open3d if imported
```

Comment:
Should it be "only import if installed"?

Reply:
Right, thank you.